Library/Formula/libfreenect2.rb

```diff
@@ -0,0 +1,143 @@
+require "formula"
+
+class Libfreenect2 < Formula
+  desc "Drivers/Example for Kinect V2"
+  homepage "https://github.com/OpenKinect/libfreenect2"
+  url "https://github.com/OpenKinect/libfreenect2/archive/master.zip"
+  sha256 "d4c3f5342965c76659f146613655c66407aee0b6b239df7bda46617de052dd14"
+
+  version "0.0.1"
+  
+  head "https://github.com/OpenKinect/libfreenect2.git"
+
+  option :universal
+
+  depends_on "libusb"
+  depends_on "nasm" => :recommended
+  depends_on "jpeg-turbo" => :recommended
+  depends_on "homebrew/science/opencv" => :recommended
+  depends_on "homebrew/versions/glfw3" => :recommended
+  depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
Please add an upstream link to the bug submission/commit containing this patch, and a short summary of why it is necessary.